Makefile.defs

```diff
@@ -79,6 +94,17 @@ else
     QFLAGS:=
 endif
 
+ifdef DARWIN
+GCCVERSION:=$(shell gcc -dumpversion | cut -c 1-3)
+ifeq ($(shell echo "$(GCCVERSION) < 4.8" | bc), 1)
```

Comment:
There should not be such restriction but currently I cannot get the static libstdc++ to work with XCode provided gcc. This also can be eliminated in the future.